.github/workflows/build.yml

```diff
@@ -182,3 +182,20 @@ jobs:
       run: |
         echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         docker push antrea/flow-aggregator:latest
+
+  build-flow-visibility-clickhouse-monitor:
+    needs: check-changes
```

Comment:
I don't think you should use the same `check-changes` step here to determine if you need to build the image.
You should look for changes in `plugins/flow-visibility/clickhouse-monitor/` instead IMO.

Reply:
Thanks Antonin for reviews! I think the changes I need to look for are 
- plugins/flow-visibility/clickhouse-monitor/
- build/images/Dockerfile.clickhouse.monitor.ubuntu
It seems `has-changes` accepts the excluded paths instead of the include ones. I try to exclude most of the paths, but not sure if it is enough, or whether there is a better way to do this.